docker/suricata/dist/suricata.yaml

```diff
@@ -1343,27 +1351,23 @@ decoder:
   # IANA assigned port 4789 is enabled.
   vxlan:
     enabled: true
-    ports: $VXLAN_PORTS # syntax: '8472, 4789'
-  # ERSPAN Type I decode support
-  erspan:
-    typeI:
-      enabled: false
+    ports: $VXLAN_PORTS # syntax: '[8472, 4789]' or '4789'.
 
   # Geneve decoder is assigned to up to 4 UDP ports. By default only the
   # IANA assigned port 6081 is enabled.
   geneve:
-    enabled: false
+    enabled: true
```

Comment:
Want to keep GENEVE disabled?

Reply:
Yes, it might be interesting to see if it catches anything.